Documentation/Blazorise.Docs.Server/Pages/_Layout.cshtml

```diff
@@ -28,6 +28,7 @@
     <link href="_content/Blazorise.Snackbar/blazorise.snackbar.min.css?v=1.0.6.0" rel="stylesheet" />
     <link href="_content/Blazorise.SpinKit/blazorise.spinkit.min.css?v=1.0.6.0" rel="stylesheet" />
     <link href="_content/Blazorise.LoadingIndicator/blazorise.loadingindicator.min.css?v=1.0.6.0" rel="stylesheet" />
+    <link href="_content/Blazorise.Animate/aos.css" rel="stylesheet" />
```

Comment:
Don't add `aos.css` manually. It will be handled by the Animate project when needed.